[ERR]: Generate PDF in Actions tab failed to run

**Where is the error?**
Actions tab

**What is written currently?**
No VM guests are running outdated hypervisor (qemu) binaries on this host.
Reading package lists...
Building dependency tree...
Reading state information...
Package libasound2 is a virtual package provided by:
  libasound2t64 1.2.11-1ubuntu0.1 (= 1.2.11-1ubuntu0.1)
  liboss4-salsa-asound2 4.2-build2020-1ubuntu3

E: Package 'libasound2' has no installation candidate
Error: Process completed with exit code 100.

**What should it be?**
Artifact ready to be downloaded

**Additional Context**
-

--- a/.github/workflows/generate-pdf.yml
+++ b/.github/workflows/generate-pdf.yml
@@ -25,7 +25,7 @@ jobs:
           sudo apt-get install -y pandoc texlive-latex-base texlive-fonts-recommended texlive-extra-utils texlive-latex-extra
           
           # Install ONLY the shared libraries needed for Chrome (DO NOT install chromium-browser)
-          sudo apt-get install -y libgbm-dev libnss3 libxss1 libasound2 libatk-bridge2.0-0 libgtk-3-0
+          sudo apt-get install -y libgbm-dev libnss3 libxss1 libasound2t64 libatk-bridge2.0-0 libgtk-3-0
 
           # Install mermaid-filter globally
           npm install --global mermaid-filter

--- a/.github/workflows/generate-pdf.yml
+++ b/.github/workflows/generate-pdf.yml
@@ -25,7 +25,7 @@ jobs:
           sudo apt-get install -y pandoc texlive-latex-base texlive-fonts-recommended texlive-extra-utils texlive-latex-extra
           
           # Install ONLY the shared libraries needed for Chrome (DO NOT install chromium-browser)
-          sudo apt-get install -y libgbm-dev libnss3 libxss1 libasound2t64 libatk-bridge2.0-0 libgtk-3-0
+          sudo apt-get install -y libgbm-dev libnss3 libxss1 libasound2t64 libatk-bridge2.0-0t64 libgtk-3-0t64
 
           # Install mermaid-filter globally
           npm install --global mermaid-filter

